src/cur-mot/common/html_2006_2013.py

```diff
@@ -0,0 +1,176 @@
+from common.html_common import (
+    add_to_docD,
+    doc_D,
+    extract_table,
+    fail_miserably,
+    handle_h1,
+    handle_h2,
+    handle_h3,
+    handle_h4,
+    handle_h5,
+    handle_h6,
+    handle_list,
+    handle_table,
+)
+from common.xml_utils import dict_to_tei
+from lxml import etree
+from bs4 import BeautifulSoup as bs
+from bs4 import NavigableString
+from bs4 import UnicodeDammit as damnit
+import inspect, json, re, sys
+
+
+
+
+def get_depth(elem):
+    if hasattr(elem, "contents") and elem.contents:
+        return max([get_depth(c) for (c) in elem.contents]) + 1
+    else:
+        return 0
+
+def parse_html_2006_2013(html, filename):
+
+    def _ck_notes(_, look_for_notes):
```

Comment:
This ```_``` is diabolical :D

Reply:
Or am I missing something? It's just a normal function parameter, better give it a name?